db/schema.rb

```diff
@@ -139,6 +139,7 @@
     t.string  "translated_name"
     t.string  "translated_description"
     t.string  "manifest_name"
+    t.string  "translated_tagline",     default: ""
```

Comment:
Why did you add a default of empty string?

Reply:
so that { translated_tagline: null } wouldn't come out in the JSON

Reply:
Well, it doesn't look like `translated_name` and `translated_description` have a default value...

Reply:
Does it hurt having a default value?

Reply:
I don't think so.